cookomatic_api/db/step.py

```diff
@@ -22,6 +22,7 @@ def save_step():
 
 class Step(ndb.Model):
     """Models a unit of instruction for cooking a Dish."""
+    number = ndb.IntegerProperty(required=True)
```

Comment:
So did you decide here to include this number or just use the implicit ordering of [`steps`](https://github.com/cookomatic/cookomatic-api/blob/master/cookomatic_api/db/dish.py#L31)?